doc/source/cluster/kubernetes/user-guides/gke-gcs-bucket.md

```diff
@@ -0,0 +1,168 @@
+
+```bash
+gcloud iam service-accounts add-iam-policy-binding my-iam-sa@my-project-id.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:my-project-id.svc.id.goog[default/my-ksa]"
+```
+
+```bash
+kubectl annotate serviceaccount my-ksa \
+    --namespace default \
+    iam.gke.io/gcp-service-account=my-iam-sa@my-project-id.iam.gserviceaccount.com
+```
+
+## Create a Google Cloud Storage Bucket and allow the Google Cloud Service Account to access it
+
+Please follow the documentation at <https://cloud.google.com/storage/docs/creating-buckets> to create a bucket using the Google Cloud Console or the `gsutil` command line tool.  
+
+For this example, we will give our principal `my-iam-sa@my-project-id.iam.gserviceaccount.com` "Storage Admin" permissions on the bucket.  You can do this in the Google Cloud Console ("Permissions" tab under "Buckets" > "Bucket Details") or with the following command:
+
+```bash
+gsutil iam ch serviceAccount:my-iam-sa@my-project-id.iam.gserviceaccount.com:roles/storage.admin gs://my-bucket
+```
+
+## Create a minimal RayCluster YAML manifest
+
+Create a file named `raycluster.yaml` with the following contents:
+
+```yaml
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-mini
+spec:
+  rayVersion: '2.5.0'
+  headGroupSpec:
+    rayStartParams:
+      dashboard-host: '0.0.0.0'
+    template:
+      spec:
+        serviceAccountName: my-ksa
+        nodeSelector:
+          iam.gke.io/gke-metadata-server-enabled: "true"
+        containers:
+        - name: ray-head
+          image: rayproject/ray:2.6.3
+          resources:
+            limits:
+              cpu: 1
+              memory: 2Gi
+            requests:
+              cpu: 500m
+              memory: 2Gi
+          ports:
+          - containerPort: 6379
+            name: gcs-server
+          - containerPort: 8265
+            name: dashboard
+          - containerPort: 10001
+            name: client
+```
+
+The key parts here are the following lines:
+
+```yaml
+      spec:
+        serviceAccountName: my-ksa
+        nodeSelector:
+          iam.gke.io/gke-metadata-server-enabled: "true"
+```
+
+These should be included in every pod spec of your Ray cluster.  In this example, we are just using a single-node cluster (1 head node and 0 worker nodes) for simplicity.
+
+## Create the RayCluster
+
+```bash
+kubectl apply -f raycluster.yaml
+```
+
+## Test GCS bucket access from the RayCluster
+
+Use `kubectl get pod` to get the name of the Ray head pod.  Then run the following command to get a shell in the Ray head pod:
+
+```bash
+kubectl exec -it raycluster-mini-head-xxxx -- /bin/bash
+```
+
+In the shell, run `pip install google-cloud-storage` to install the Google Cloud Storage Python client library.  Then run the following Python code to test access to the bucket:
```

Comment:
```suggestion
In the shell, run `pip install google-cloud-storage` to install the Google Cloud Storage Python client library. Then run the following Python code to test access to the bucket:
```

Reply:
I don't think this makes a difference in the rendered markdown.  Some people were taught to use two spaces after a period and it's a hard habit to break unfortunately... I'd propose ignoring it until we install a linter that automatically formats it and enforces it